### PR TITLE
[rbp] update README

### PR DESCRIPTION
**Pull request by FernetMenta**
see title

### CODE REVIEW

**Comment by popcornmix** [comment]
+1
The new method is simpler and works for me.
Just a note, this builds for Pi1, which does work on Pi2, but will not be as efficient as an armv7/neon build.
It also builds a debug build.

**Comment by wsnipex** [comment]
all those CFLAGS are not needed, configure sets them already here https://github.com/xbmc/xbmc/blob/master/tools/depends/configure.in#L308

**Comment by FernetMenta** [comment]
updated @popcornmix @wsnipex ok?

**Comment by wsnipex** [comment]
well the LDFLAGS and many on the configure switches(they belong to kodi, not depends) are superflous as well.  https://github.com/xbmc/xbmc/blob/master/tools/depends/README#L26-L27 should be enough. Or is anything missing there?

File: docs/README.raspberrypi [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
+       --disable-vdpau --disable-projectm --disable-rsxs --disable-fishbmc \
+       --disable-alsa --disable-solarwinds --disable-euphoria \
+       --disable-plasma
+
+    make
+    cd ../..
+
+    CONFIG_EXTRA="--with-platform=raspberry-pi \
+       --enable-libcec --enable-player=omxplayer \
+       --disable-sdl --disable-x11 --disable-xrandr --disable-openmax \
```
Comment (wsnipex) [inline docs/README.raspberrypi]: --disable-sdl does not exist anymore, it was superseded by joystick. I'd not disable joystick though(see next line)
Alsa might be needed as well? I'm thinking USB soundcards.

Comment (popcornmix) [inline docs/README.raspberrypi]: Yes, alsa is useful for USB/I2S sound cards.

**Comment by popcornmix** [comment]
AFAICT none of the first set of configure options are recognised:
>configure: WARNING: unrecognized options: --disable-debug, --disable-gl, --enable-gles, --enable-airplay, --enable-airtunes, --enable-libcec, --enable-player, --disable-sdl, --disable-x11, --disable-xrandr, --disable-openmax, --disable-optical-drive, --disable-dvdcss, --disable-joystick, --disable-crystalhd, --disable-vtbdecoder, --disable-vaapi, --disable-vdpau, --disable-projectm, --disable-rsxs, --disable-fishbmc, --disable-alsa, --disable-solarwinds, --disable-euphoria, --disable-plasma

**Comment by FernetMenta** [comment]
updated

**Comment by wsnipex** [comment]
I'm still unsure about disabling joystick, but I'll leave that to @popcornmix to decide.

**Comment by FernetMenta** [comment]
this is for developers not for users. developers dealing with joysticks on the PI are the minority.

**Comment by popcornmix** [comment]
I've tested latest instructions and it built okay.
I'm not too fussed about whether joysticks are enabled/disabled.

**Pull request merged**